ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -102,7 +102,7 @@
 ipafiles_checks = ["IPAFileNSSDBCheck", "IPAFileCheck", "TomcatFileCheck"]
 dogtag_checks = ["DogtagCertsConfigCheck", "DogtagCertsConnectivityCheck"]
 iparoles_checks = ["IPACRLManagerCheck", "IPARenewalMasterCheck"]
-replication_checks = ["ReplicationCheck"]
+replication_checks = ["ReplicationConflictCheck"]
```

Comment:
This line should not be changed upstream. Please see commit https://pagure.io/freeipa/c/d238fb4f2bb6525367a73c42a8a238843d5c8b43 and ticket https://pagure.io/freeipa/issue/8430 which explains why the replication_checks variable has "ReplicationCheck" value: ipa-healthcheck 0.6 is using the check defined by 389-ds.
If in RHEL an older version is used (like 0.4), the ipatests pkg must be built without the commit d238fb4f2bb6525367a73c42a8a238843d5c8b43.